src/domain/feedback/mutation.ts

```diff
@@ -0,0 +1,27 @@
+// eslint-disable-next-line import/no-named-as-default
+import gql from 'graphql-tag';
+
+export const MUTATION_FEEDBACK = gql`
+  mutation PostFeedback($input: FeedbackInput!) {
+    postFeedback(input: $input)
+      @rest(
+        type: "Feedback"
+        path: "/feedback/"
+        method: "POST"
+        bodyKey: "input"
```

Comment:
Seems like bodyKey is "input" by default according to documentation. Is this intetionally added here? I mean it doesn't to have it but could also be omitted?

Reply:
I have it here just in case that the default value changes at some point. But could also be omitted as you said.

Reply:
It doesn't hurt to keep it :)